File: src/main/java/com/f2prateek/bee/BeeApp.java
package com.f2prateek.bee;

import android.app.Application;
import timber.log.Timber;

public class BeeApp extends Application {
  @Override public void onCreate() {
    super.onCreate();

    if (BuildConfig.DEBUG) {
      Timber.plant(new Timber.DebugTree());
    }
  }
}

File: src/main/java/com/f2prateek/bee/BeeApp.kt
package com.f2prateek.bee

import android.app.Application
import timber.log.Timber

class BeeApp : Application() {
    override fun onCreate() {
        super.onCreate()

        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
    }
}
